### DISCUSSION TITLE
Smartapbanner on top and CLS problem

### DISCUSSION THREAD

**Discussion by michalidesko** [ain/smartbanner.js#595] [General]
When I started using smartbanner, I had a problem with CLS - https://web.dev/cls/    I know that I can turn off the positioning - `<meta name="smartbanner:disable-positioning" content="true">` , but then I don't know how to solve the content overlapping nicely. Can anyone suggest a nice solution how to solve this?

**Comment by ain** [comment]
Could it be that you're experiencing what has been described in #279 before?

**Comment by michalidesko** [comment]
I do not think so. Everything works, the only problem is that a place is not reserved for the banner when the conditions for display are met (for example: it was not previously closed) Then it jumps. What makes the problem with CLS

**Comment by AleksandrKarashev** [comment]
same
